.circleci/config.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   test:
     docker:
       # specify the version you desire here
-      - image: circleci/ruby:2.7-buster
+      - image: circleci/ruby:2.7.0-node
```

Comment:
can you use 2.7.1, 2.7.0 has some security vulnerabilities.

Reply:
done.